### ISSUE TITLE
Horse inventory not implemented

### ISSUE THREAD

**Issue by rtm516** [GeyserMC/Geyser#639]
**Describe the bug**
The horse inventory isn't implemented

**To Reproduce**
1. Get on horse
2. Press E to open inventory

**Expected behavior**
The inventory gets opened when requested

**Screenshots / Videos**
N/A

**Server Version**
git-Paper-155 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)

**Geyser Version**
Master https://github.com/GeyserMC/Geyser/commit/3d0dbff87b002a2a7409cb771ddc07c8e880f070

**Minecraft: Bedrock Edition Version**
Win10 1.14.60

**Additional Context**
This is the packet sent from the client when the inventory is attempted to be opened.
```
[SERVER BOUND]  -  InteractPacket(action=OPEN_INVENTORY, runtimeEntityId=3, mousePosition=null)
```
Looks like java uses a separate packet for sending it https://wiki.vg/Protocol#Open_Horse_Window

**Comment by maurotramonti** [comment]
Are you working on this?

**Comment by AnonAlpha** [comment]
> Are you working on this?

No need to urge them, they're working on the thing they think is the most needed

**Comment by maurotramonti** [comment]
Ah ok, thank you for the info.

**Comment by DoctorMacc** [comment]
It's likely that this will not be implemented until after 1.16 since hopefully we will shift our focus to that soon.
